src/tests/work-item/work-item-list/work-item-timeStamp.spec.js

```diff
@@ -34,16 +34,18 @@ describe('Work item list', function () {
     expect(detailPage.getCreatedtime()).toBe('17 minutes ago');
     page.workItemByURLId("id2");
     expect(detailPage.getCreatedtime()).toBe('8 hours ago');
-    page.workItemByURLId("id3");
-    expect(detailPage.getCreatedtime()).toBe('3 days ago');
-    page.workItemByURLId("id4");
-    expect(detailPage.getCreatedtime()).toBe('15 days ago');
-    page.workItemByURLId("id5");
-    expect(detailPage.getCreatedtime()).toBe('25 days ago');
-    page.workItemByURLId("id6");
-    expect(detailPage.getCreatedtime()).toBe('a month ago');
-    page.workItemByURLId("id7");
-    expect(detailPage.getCreatedtime()).not.toBe('a year ago');
+
+    // So many tests in one 'it' cause spec timeout some time
+    // page.workItemByURLId("id3");
```

Comment:
Such  tests have been written on other files also
https://github.com/fabric8io/fabric8-planner/blob/master/src/tests/work-item/work-item-list/displayAndUpdateWorkItemDetails.spec.js#L405

We need a separate PR to fix this.